### PR TITLE
Fix default value bug

### LINKED ISSUE #1368: It need to judge the default value is the same as the given type when create schema
image

### PR DESCRIPTION
**Pull request by laura-ding**
colse #1421, close #1368

### CODE REVIEW

File: src/graph/InsertVertexExecutor.cpp [hunk 1]
```diff
@@ -48,14 +48,14 @@ Status InsertVertexExecutor::check() {
         auto *tagName = item->tagName();
         auto tagStatus = ectx()->schemaManager()->toTagID(spaceId_, *tagName);
         if (!tagStatus.ok()) {
-            LOG(ERROR) << "No schema found for " << tagName;
+            LOG(ERROR) << "No schema found for " << *tagName;
```
Comment (darionyaphet) [inline src/graph/InsertVertexExecutor.cpp]: May bad orz

File: src/graph/test/SchemaTest.cpp [hunk 2]
```diff
@@ -46,6 +46,36 @@ TEST_F(SchemaTest, TestComment) {
     }
 }
 
+TEST_F(SchemaTest, TestDefaultValue) {
+    auto client = gEnv->getClient();
```
Comment (darionyaphet) [inline src/graph/test/SchemaTest.cpp]: have a test case about default value ?

Comment (laura-ding) [reply src/graph/test/SchemaTest.cpp]: data_test has include it

File: src/graph/test/SchemaTest.cpp [hunk 3]
```diff
@@ -46,6 +46,36 @@ TEST_F(SchemaTest, TestComment) {
...
+        std::string cmd = "CREATE TAG default_tag(name string, age int DEFAULT \"10\")";
+        auto code = client->execute(cmd, resp);
+        ASSERT_EQ(cpp2::ErrorCode::E_EXECUTION_ERROR, code);
+    }
+    {
+        cpp2::ExecutionResponse resp;
+        std::string cmd = "CREATE TAG default_tag(name string  DEFAULT \"\", "
+                          "age int DEFAULT \"10\")";
+        auto code = client->execute(cmd, resp);
+        ASSERT_EQ(cpp2::ErrorCode::E_EXECUTION_ERROR, code);
```
Comment (darionyaphet) [inline src/graph/test/SchemaTest.cpp]: duplicate with previous

**Pull request merged**